subsys/bluetooth/audio/has.c

```diff
@@ -32,8 +32,33 @@ LOG_MODULE_REGISTER(bt_has, CONFIG_BT_HAS_LOG_LEVEL);
  */
 #define BT_HAS_MAX_CONN MIN(CONFIG_BT_MAX_CONN, CONFIG_BT_MAX_PAIRED)
 
+#define BIT_CHANGED(_new_value, _old_value) ((_new_value) ^ (_old_value))
+#define FEATURE_DEVICE_TYPE_UNCHANGED(_new_type) \
+	!(BIT_CHANGED((_new_type) & BT_HAS_FEAT_HEARING_AID_TYPE_LSB, \
+			      (has.features & BT_HAS_FEAT_HEARING_AID_TYPE_LSB))) && \
+	!(BIT_CHANGED((_new_type) & BT_HAS_FEAT_HEARING_AID_TYPE_MSB, \
+			      (has.features & BT_HAS_FEAT_HEARING_AID_TYPE_MSB)))
```

Comment:
Instead of using the LSB and MSB, could you not use `BT_HAS_FEAT_HEARING_AID_TYPE_MASK`?

Reply:
Oh yes I can, this one escaped my gaze, much better indeed.